### PR TITLE
✨ Add releasedOf view

### LINKED ISSUE #161: Add vester releasedOf view
For farming purpose

### PR DESCRIPTION
**Pull request by Bal7hazar**
Fix #161

### CODE REVIEW

File: src/vesting/vester.cairo [hunk 1]
```diff
@@ -251,6 +251,23 @@ func releasableOf{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_pt
...
+    //   Return
+    // Implicit args:
+    //   syscall_ptr(felt*)
+    //   pedersen_ptr(HashBuiltin*)
+    //   range_check_ptr
+    // Explicit args:
+    //   account(felt): Account address
+    // Returns:
+    //   amount(Uint256): Total amount released
+    return CarbonableVester.released_of(account=account);
```
Comment (cloudvenger) [inline src/vesting/vester.cairo:268]: Do we want that only the account and Admin can have access to this function ? (which Access Control we want to apply ?)

**Review by cloudvenger — COMMENTED** [review]
clean 👍

File: src/vesting/vester.cairo (hunk 1 above)
Comment (Bal7hazar) [reply src/vesting/vester.cairo:268]: This is a view, so no access control required imo

Comment (cloudvenger) [reply src/vesting/vester.cairo:268]: 👍

**Review by cloudvenger — APPROVED** [review]

**Pull request merged**